certora/specs/wadRayMath.spec

```diff
@@ -208,39 +209,39 @@ rule rayToWadLiveness(uint256 a) {
 rule wadToRaySafety(uint256 a) {
     uint res = wadToRay(a);
 
-    assert res == a * WADTORAY();
+    assert to_mathint(res) == a * WADTORAY();
 }
 
 rule wadToRayLiveness(uint256 a) {
     wadToRay@withrevert(a);
 
-    assert lastReverted <=> a * WADTORAY() > 2^256;
+    assert lastReverted <=> a * WADTORAY() > UINT_LIMIT();
```

Comment:
Both are equivalent (because `WADTORAY()` is not a factor of `UINT_LIMIT()`) but I agree that it would be clearer as you suggested. I'll open a PR on top to not lose the approvals